<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Rec</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_packets">
       <attribute name="title">
        <string>Packets</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_loadOP">
              <property name="text">
               <string>Load OPcode: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_opcodes"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_opcode">
              <property name="text">
               <string>OP code: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_opcode">
              <property name="maxLength">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_arg1">
              <property name="text">
               <string>Arg1: </string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_arg1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>00</string>
              </property>
              <property name="maxLength">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_arg2">
              <property name="text">
               <string>Arg2: </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_arg2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>00</string>
              </property>
              <property name="maxLength">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_genPacket">
              <property name="text">
               <string>Create Packet</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_chksum">
              <property name="text">
               <string>Checksum: </string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_chk">
              <property name="maxLength">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label_packetText">
              <property name="text">
               <string>Packet: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_packet"/>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_sendPacket">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Send Packet!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Previous Packets: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_packetHistory"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_packets"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_serial">
       <attribute name="title">
        <string>Serial Port</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_serial">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>281</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Connect</string>
        </property>
        <widget class="QComboBox" name="comboBox_serialList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_serialConnect">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>60</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_serialRefreshList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_console">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowser_console"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
